test/built-ins/String/prototype/normalize/return-abrupt-from-form.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 21.1.3.12
+description: >
+  Returns abrupt from ToString(form)
```

Comment:
this is a decision I've made from a previous PR discussion.

I avoid merging tests where I can isolate new ES6 features, as the ones using Symbol.